src/test/groovy/org/prebid/server/functional/model/request/auction/DsaTransparency.groovy

```diff
@@ -7,7 +7,7 @@ import org.prebid.server.functional.util.PBSUtils
 class DsaTransparency {
 
     String domain
-    List<Integer> params
+    List<Integer> dsaparams
```

Comment:
Please follow Java code style and use @JsonProperty or use Jackson naming strategy